Orange/classification/majority.py

```diff
@@ -82,3 +82,5 @@ def predict(self, X):
 
     def __str__(self):
         return 'ConstantModel {}'.format(self.dist)
+
+MajorityLearner.__returns__ = ConstantModel
```

Comment:
Aww, you are so considerate! Why didn't you use the same approach in naive_bayes.py below? :smiley: